Month1/Week2/Section2/Month1-Week2-Section2/Spotify/PlayList.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include "Song.h"
+
+class FPlayList {
```

Comment:
Also, "playlist" is just one word hehe, it should be FPlaylist instead of FPlayList

Reply:
all the PlayList changed for Playlist